rest/data/patch.go

```diff
@@ -39,11 +39,18 @@ type DBPatchConnector struct{}
 // FindPatchesByProject uses the service layer's patches type to query the backing database for
 // the patches.
 func (pc *DBPatchConnector) FindPatchesByProject(projectId string, ts time.Time, limit int) ([]restModel.APIPatch, error) {
-	patches, err := patch.Find(patch.PatchesByProject(projectId, ts, limit))
+	apiPatches := []restModel.APIPatch{}
+	id, err := model.GetIdForProject(projectId)
+	if id == "" {
```

Comment:
this will also be true in the error case, so we should check the user first

Reply:
done